### PR TITLE
MultiSense s21

### PR DESCRIPTION
**Pull request by fabienrohrer**
Dual PR: https://github.com/omichel/webots/pull/6658

Result: https://www.cyberbotics.com/doc/automobile/camera-sensors?version=enhancement-multisense-s21#multisense-s21

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: automobile/camera-sensors.md [hunk 1]
```diff
@@ -16,3 +16,70 @@ If a `Recognition` node is added to the camera, the camera can return not only t
 ![smart_camera.png](images/smart_camera.png)
 
 %end
+
```
Comment (DavidMansolino) [inline automobile/camera-sensors.md]: The beginning of this page should probably be update:
> No specific model of camera have been modeled,

**Comment by fabienrohrer** [comment]
👍

**Comment by fabienrohrer** [comment]
Thanks all ;-)

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**